replicate-to-sql-with-generic-events/lib/index.js

```diff
@@ -0,0 +1,60 @@
+const { PostgresWrapper, pgConfigDocker } = require('./postgres');
+
+class CorePlugin {
+  constructor() {
+    this.context = null;
+    this.config = {};
+    this.pipes = {
+      'generic:document:afterWrite': 'afterWrite',
+      'generic:document:afterDelete': 'afterDelete'
+    };
+  }
+
+  init(customConfig, context) {
+    this.config = Object.assign(this.config, customConfig);
+    this.context = context;
+    this.pg = this.initConnection();
+  }
+
+  initConnection() {
+    return new PostgresWrapper(pgConfigDocker);
+  }
+
+  getProperties(doc) {
+    const properties = Object.keys(doc._source).reduce((object, key) => {
```

Comment:
I would like to question as why this has been put in the kuzzle._source property ?
I have seen [some places](https://github.com/kuzzleio/kuzzle-how-to/blob/master/sync-data-to-another-database/plugin/lib/services/exporter.js) where we make the same type of operation to delete this _kuzzle_info property, shall it be placed at the root level of the documents reponse ?

Reply:
Before Kuzzle metadata where exposed alongside the `_source` inside a `meta` property:
```
// response
{
  _id: 'some id',
  _source: { /* document body */
 meta: { /* kuzzle metadata */ }
}
```
But it was confusing for the user because the metadata where stored in the `_kuzzle_info` property inside the document body.  
For example when someone wanted to sort the documents of a search by creation date, they didn't know that they should use the filed `_kuzzle_info.createdAt`

Reply:
why not simply renaming the meta property into _kuzzle_info directly at the root document ?
to avoid altering the source document witch will may be used by another piece of code

IMHO I think that is extra work that can be avoided

Reply:
Hmm yes it could be a good idea. I'm just wondering if the users will understand that these metadata are in fact stored in the body and so they can use them in the search queries, in aggregations, etc  
We could discuss this point during the next product workshop

Reply:
The name is not the only reason why we removed the meta object and replaced it by a raw _kuzzle_info one inside the _source property of the document.

This all came from this use case: "I want to be able to search modified documents by author, how should I write my query?"